publishing-kube-rules.yaml

```diff
@@ -0,0 +1,50 @@
+skip-godeps: true
+rules:
+- destination: kubernetes-code-generator
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: staging/src/k8s.io/code-generator
+- destination: kubernetes-apimachinery
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: staging/src/k8s.io/apimechinery
+- destination: kubernetes-api
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: staging/src/k8s.io/api
+- destination: kubernetes-client-go
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: staging/src/k8s.io/client-go
+- destination: kubernetes-metrics
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: staging/src/k8s.io/metrics
+- destination: kubernetes-apiserver
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: staging/src/k8s.io/apiserver
+- destination: kubernetes-aggregator
```

Comment:
It's kubernetes-kube-aggregator: https://github.com/openshift/kubernetes-kube-aggregator